### PR TITLE
Types for undertaker-registry@1.0

### PR DESCRIPTION
**Pull request by GiedriusGrabauskas**
Package: https://npmjs.com/package/undertaker-registry

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/undertaker-registry/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * Returns an object listing all tasks in the registry.
+     * Necessary to override if the get method is overridden for custom task storage.
+     * Custom registries can override this when when inheriting from this default registry.
+     */
+    tasks(): { [taskName: string]: (...args: any[]) => any };
+}
+
+declare namespace UndertakerRegistry { }
```
Comment (sheetalkamat) [inline types/undertaker-registry/index.d.ts]: why do you have this? I don't see any documentation why this is needed

Comment (GiedriusGrabauskas) [inline types/undertaker-registry/index.d.ts]: Without declared namespace this line:
```ts
import * as UndertakerRegistry from "undertaker-registry";
```
Throw typescript error:
```cmd
Module '"types/undertaker-registry/index"' resolves to a non-module entity and cannot be imported using this construct.
```

Comment (sheetalkamat) [inline types/undertaker-registry/index.d.ts]: http://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

Comment (GiedriusGrabauskas) [inline types/undertaker-registry/index.d.ts]: https://github.com/DefinitelyTyped/DefinitelyTyped/#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Comment (GiedriusGrabauskas) [inline types/undertaker-registry/index.d.ts]: This import:
```ts
import * as UndertakerRegistry from "undertaker-registry";
```
Is the same as:
```ts
const UndertakerRegistry  = require("undertaker-registry");
```

**Pull request merged**